tests/Writing/SetPropertyTypesTest.php

```diff
@@ -185,13 +185,13 @@ public function testCreateValueBoolean()
         $this->assertInstanceOf(PropertyInterface::class, $value);
         $this->assertEquals(PropertyType::BOOLEAN, $value->getType(), 'wrong type');
         $this->assertTrue($value->getBoolean(), 'boolean not true');
-        $this->assertTrue($value->getString() == true, 'wrong string value'); //boolean converted to string must be true
+        $this->assertEquals($value->getString(), true, 'wrong string value'); //boolean converted to string must be true
```

Comment:
i think this should be the other way rount, with `true` being the first value. i guess assertTrue would check that the string is a boolean, which is not the case.

Reply:
Yep, you're right. In `assertEquals`, the first parameter is the `$expected`, in our case `true`, and the second one is the `$actual`, in our case `$value->getString()`. Gonna change it and search for similar.